### DISCUSSION TITLE
2021-10-06 Weekly Meeting 24 Agenda

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#103] [Weekly Meetings]
Weekly Meeting 24 is happening on Wednesday October 6, 2021 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - Let's review our backlog!
     - [OMI Github strategy](https://github.com/omigroup/website/issues/34)
   - OMI Website
   - Vket Community Booth
   - glTF subgroup
   - Community field trip update

Add ons: 
   - As you see things of interest in discord, a convo or a demo, take a screenshot and drop it into the comments of the next agenda.

**Comment by mrmetaverse** [comment]
gltf subgroup has a vote scheduled for tomorrow at 5:30 EDT

**Comment by mrmetaverse** [comment]
[OMI_Audio_Emitter](https://github.com/omigroup/gltf-extensions/pull/1) seeking at least one more implementer before we can consider the work "ready".

**Comment by mrmetaverse** [comment]
TL;DR: (1) Paste notes and thoughts during meetings in the existing meetings comments. (2) Keep an eye out for interesting screenshots, demos, or conversations in our community and share them on the following weeks agenda comments (as items to potentially discuss during the next meeting).

We want to approach sharing items, taking notes, and leaving breadcrumbs for those who couldn't attend by leveraging the github agendas comments feature. Instead of creating a separate note, lets keep notes here in the agenda comments. We want to make it easier for folks who couldn't attend to get up to speed. 

If you see something cool happening that someone is showcasing in discord on #hack-chat for example, please take a screenshot and paste it in the comments of _the next meetings agenda_. This can help ensure innovation is getting lost in discord threads, peoples accomplishments are shared early and often, and may even spark conversation in our meetings that we may otherwise have missed.

**Comment by mrmetaverse** [comment]
Please review our new documented ["OMI Github Strategy"](https://github.com/omigroup/website/issues/34) discussion.
